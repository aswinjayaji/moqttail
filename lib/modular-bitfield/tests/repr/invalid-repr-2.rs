use modular_bitfield::prelude::*;

#[bitfield]
#[cfg_attr(not(feature = "unknown"), repr(invalid))]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
