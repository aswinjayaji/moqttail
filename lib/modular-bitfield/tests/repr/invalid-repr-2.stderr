error[E0552]: unrecognized representation hint
 --> $DIR/invalid-repr-2.rs:4:43
  |
4 | #[cfg_attr(not(feature = "unknown"), repr(invalid))]
  |                                           ^^^^^^^
